src/styles/antd-overrides/button.scss

```diff
@@ -80,6 +80,7 @@
 
 .ant-btn-link {
   color: var(--text-action-primary);
+  border: none;
```

Comment:
@johnnyd-eth I think the hover state might still has a border (haven't checked though)